runtime/doc/lua.txt

```diff
@@ -2391,6 +2400,8 @@ vim.validate({opt})                                           *vim.validate()*
 ==============================================================================
 Lua module: vim.loader                                            *vim.loader*
 
+Loader:loadfile()                                          *Loader:loadfile()*
+
```

Comment:
`@nodoc?`

Reply:
It's already got `@private`. The issue is it has `-- luacheck: ignore 312`, and the `--` breaks away the doc block from the function.

Reply:
Fixed now.